### PR TITLE
feat: add annotations form adaption support

### LINKED ISSUE #14: 添加模型元数据表单定义的支持。
跟友联插件那样

### PR DESCRIPTION
**Pull request by LIlGG**
#### What type of PR is this?

/kind feature

#### Which issue(s) this PR fixes:

Fixes #14

#### What this PR does / why we need it:

添加模型元数据表单定义的支持。由于引入了 annotationsForm ，因此 Halo 版本依赖升级为 >=2.8.0

image

image

#### Does this PR introduce a user-facing change?
```release-note
添加模型元数据表单定义的支持。
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -345,3 +345,11 @@ List<[#PhotoVo](#photovo)>
...
 ```
 
+### Annotations 元数据适配
+
+根据 Halo 的[元数据表单定义文档](https://docs.halo.run/developer-guide/annotations-form/)和[模型元数据文档](https://docs.halo.run/developer-guide/theme/annotations)，Halo 支持为部分模型的表单添加元数据表单，此插件同样适配了此功能，如果你作为主题开发者，需要为链接或者链接分组添加额外的字段，可以参考上述文档并结合下面的 TargetRef 列表进行适配。
+
+| 对应模型   | group            | kind       |
+| ---------- | ---------------- | ---------- |
+| 链接       | core.halo.run | Photo       |
+| 链接分组 | core.halo.run | PhotoGroup |
```
Comment (ruibaby) [inline README.md:355]: 描述不对

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Review by guqing — COMMENTED** [review]
/lgtm

image

功能符合预期👍

**Review by JohnNiang — APPROVED** [review]
/lgtm

**Review by ruibaby — APPROVED** [review]

**Pull request merged**